build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -9,6 +9,14 @@
   <Import Project="$(_SourceTopDir)\Configuration.props" />
   <Import Project="mono-runtimes.props" />
   <Import Project="mono-runtimes.projitems" />
+  <ItemGroup>
+      <_BclProfileItems Include="$(_MonoPath)\mcs\class\lib\monodroid\mscorlib.dll"/>
+      <_BclProfileItems Include="$(_MonoPath)\mcs\class\lib\monodroid\System.dll"/>
+      <_BclProfileItems Include="$(_MonoPath)\mcs\class\lib\monodroid\System.Core.dll"/>
+      <_BclProfileItems Include="$(_MonoPath)\mcs\class\lib\monodroid\System.Net.Http.dll"/>
+      <_BclProfileItems Include="$(_MonoPath)\mcs\class\lib\monodroid\System.Xml.dll"/>
+      <_BclProfileItems Include="$(_MonoPath)\mcs\class\lib\monodroid\Mono.Data.Sqlite.dll"/>
```

Comment:
We shouldn't require `Mono.Data.Sqlite`, as this will need to be built separately by xamarin-android.